README.md

```diff
@@ -48,25 +56,7 @@ This metadata is required by the SignalFx Agent to set global dimensions such as
 
 See [here](https://devcenter.heroku.com/articles/dyno-metadata) for more information.
 
-**Overriding the default Agent config**
+### Overriding the default Agent config
 
 The default [SignalFx Agent config](./setup/config.yaml) will be overridden if a config is provided in `signalfx/agent.yaml`
-in the root of the Heroku project directory. In such cases, it is recommended that the following defaults are retained.
-
-```
-# get access token
-signalFxAccessToken: {"#from": "env:SFX_TOKEN"}
-
-# dimensionalizes metadata exposed in dynos
```

Comment:
Since we're removing this note from the doc, I think we should add a comment in the default config stating why these dimensions are being added by default - 1) the heroku metadata monitor put properties onto the dyno_id dimension, so those properties would only show up on agent MTSes, only if the dyno_id dimension exists and these dimensions are synced by the Heroku collector so having them on MTSes from the agent would be useful to group by or filter by one of attributes.

Reply:
@asuresh4 How about just adding a link? PR is updated.